packages/lit-dev-content/site/tutorials/content/pointer-events/04.md

```diff
@@ -0,0 +1,20 @@
+Another powerful feature of Pointer Events is the ability to control the brower's native touch behavior with the use of the [`touch-action`](https://developer.mozilla.org/en-US/docs/Web/CSS/touch-action) css property.
+
+A pointer will fire events until the browser determines that a native behavior should occur, such as scrolling or zooming, at which point the browser will fire a final `pointercancel` event for the interaction.
+
+**Touch Action Settings**
+
+* `auto`
+  * This is the default setting. This means that the browser will fire events until scrolling, zooming, or browser specific behavior such as "double tap to zoom" occurs.
+* `none`
+  * With this setting, the browser will always fire events, and will prevent all touch behaviors like scrolling and zooming.
+* `pan-left`, `pan-right`, `pan-x`, `pan-up`, `pan-down`, and `pan-y`
+  * These settings will let the browser scroll in the specified direction.
+  * `pan-x` is shorthand for `pan-left pan-right`
+  * `pan-y` is shorthand for `pan-up pan-down`
+* `pinch-zoom`
+  * This setting will let the browser perform pinch to zoom behaviors, but not scrolling or "double tap to zoom".
+* `manipulation`
+  * This setting is equivalent to `pan-x pan-y pinch-zoom`, only disabling "double tap to zoom" and other browser-specific behaviors.
+
+Try using a touchscreen with this example to see how the different settings for `touch-action` control event firing and browser behavior.
```

Comment:
Add a congratulations or you're done page after this that recaps the knowledge learned and points to docs or readings that the user should probably logically follow up with next. e.g. event handling in lit, gestures articles / docs in MDN, setting event options in lit, shadow dom and events docs and maybe this article that I'm in love with

https://pm.dartus.fr/blog/a-complete-guide-on-shadow-dom-and-event-propagation/